### ISSUE TITLE
Can't install under strawberry perl 5.30.0

### ISSUE THREAD

**Issue by zdm** [msgpack/msgpack-perl#42]
```
cpanm (App::cpanminus) 1.7044 on perl 5.030000 built for MSWin32-x64-multi-thread
Work directory is C:\Users\zdm\AppData\Local\Temp\.cpanm/work/1558802524.364
You have make D:\devel\perl\c\bin\gmake.exe
You have LWP 6.39
You have C:\WINDOWS\system32\tar.exe, D:\devel\msys2\usr\bin\gzip.exe and D:\devel\msys2\usr\bin\bzip2.exe
Searching Data::MessagePack on mirror https://cpan.metacpan.org ...
Downloading index file https://cpan.metacpan.org/modules/02packages.details.txt.gz ...
--> Working on Data::MessagePack
Fetching https://cpan.metacpan.org/authors/id/S/SY/SYOHEX/Data-MessagePack-1.00.tar.gz ... OK
Unpacking Data-MessagePack-1.00.tar.gz
x Data-MessagePack-1.00/
x Data-MessagePack-1.00/benchmark/
x Data-MessagePack-1.00/benchmark/serialize.pl
x Data-MessagePack-1.00/benchmark/data.pl
x Data-MessagePack-1.00/benchmark/size.pl
x Data-MessagePack-1.00/benchmark/deserialize.pl
x Data-MessagePack-1.00/include/
x Data-MessagePack-1.00/include/msgpack/
x Data-MessagePack-1.00/include/msgpack/pack_define.h
x Data-MessagePack-1.00/include/msgpack/unpack_define.h
x Data-MessagePack-1.00/include/msgpack/sysdep.h
x Data-MessagePack-1.00/include/msgpack/unpack.h
x Data-MessagePack-1.00/include/msgpack/pack.h
x Data-MessagePack-1.00/include/msgpack/predef.h
x Data-MessagePack-1.00/include/msgpack/pack_template.h
x Data-MessagePack-1.00/include/msgpack/predef/
x Data-MessagePack-1.00/include/msgpack/predef/os/
x Data-MessagePack-1.00/include/msgpack/predef/os/android.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd.h
x Data-MessagePack-1.00/include/msgpack/predef/os/ios.h
x Data-MessagePack-1.00/include/msgpack/predef/os/hpux.h
x Data-MessagePack-1.00/include/msgpack/predef/os/unix.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/open.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/free.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/dragonfly.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/bsdi.h
x Data-MessagePack-1.00/include/msgpack/predef/os/bsd/net.h
x Data-MessagePack-1.00/include/msgpack/predef/os/beos.h
x Data-MessagePack-1.00/include/msgpack/predef/os/vms.h
x Data-MessagePack-1.00/include/msgpack/predef/os/irix.h
x Data-MessagePack-1.00/include/msgpack/predef/os/haiku.h
x Data-MessagePack-1.00/include/msgpack/predef/os/macos.h
x Data-MessagePack-1.00/include/msgpack/predef/os/aix.h
x Data-MessagePack-1.00/include/msgpack/predef/os/qnxnto.h
x Data-MessagePack-1.00/include/msgpack/predef/os/cygwin.h
x Data-MessagePack-1.00/include/msgpack/predef/os/amigaos.h
x Data-MessagePack-1.00/include/msgpack/predef/os/os400.h
x Data-MessagePack-1.00/include/msgpack/predef/os/windows.h
x Data-MessagePack-1.00/include/msgpack/predef/os/linux.h
x Data-MessagePack-1.00/include/msgpack/predef/os/solaris.h
x Data-MessagePack-1.00/include/msgpack/predef/version_number.h
x Data-MessagePack-1.00/include/msgpack/predef/library/
x Data-MessagePack-1.00/include/msgpack/predef/library/c.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std.h
x Data-MessagePack-1.00/include/msgpack/predef/library/c/
x Data-MessagePack-1.00/include/msgpack/predef/library/c/gnu.h
x Data-MessagePack-1.00/include/msgpack/predef/library/c/vms.h
x Data-MessagePack-1.00/include/msgpack/predef/library/c/zos.h
x Data-MessagePack-1.00/include/msgpack/predef/library/c/_prefix.h
x Data-MessagePack-1.00/include/msgpack/predef/library/c/uc.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/
x Data-MessagePack-1.00/include/msgpack/predef/library/std/sgi.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/dinkumware.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/roguewave.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/stdcpp3.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/_prefix.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/libcomo.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/vacpp.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/modena.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/stlport.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/msl.h
x Data-MessagePack-1.00/include/msgpack/predef/library/std/cxx.h
x Data-MessagePack-1.00/include/msgpack/predef/version.h
x Data-MessagePack-1.00/include/msgpack/predef/platform.h
x Data-MessagePack-1.00/include/msgpack/predef/language/
x Data-MessagePack-1.00/include/msgpack/predef/language/objc.h
x Data-MessagePack-1.00/include/msgpack/predef/language/stdc.h
x Data-MessagePack-1.00/include/msgpack/predef/language/stdcpp.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/
x Data-MessagePack-1.00/include/msgpack/predef/compiler/sunpro.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/digitalmars.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/gcc.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/iar.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/sgi_mipspro.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/greenhills.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/diab.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/gcc_xml.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/borland.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/microtec.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/watcom.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/compaq.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/mpw.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/pgi.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/ibm.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/kai.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/dignus.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/palm.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/metaware.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/comeau.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/intel.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/llvm.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/edg.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/ekopath.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/tendra.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/visualc.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/hp_acc.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/metrowerks.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler/clang.h
x Data-MessagePack-1.00/include/msgpack/predef/compiler.h
x Data-MessagePack-1.00/include/msgpack/predef/language.h
x Data-MessagePack-1.00/include/msgpack/predef/make.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/
x Data-MessagePack-1.00/include/msgpack/predef/architecture/pyramid.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/sys370.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/sys390.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/z.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/mips.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/x86.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/rs6k.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/parisc.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/ia64.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/sparc.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/x86/
x Data-MessagePack-1.00/include/msgpack/predef/architecture/x86/64.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/x86/32.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/blackfin.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/superh.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/alpha.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/ppc.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/m68k.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/convex.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture/arm.h
x Data-MessagePack-1.00/include/msgpack/predef/other/
x Data-MessagePack-1.00/include/msgpack/predef/other/endian.h
x Data-MessagePack-1.00/include/msgpack/predef/other.h
x Data-MessagePack-1.00/include/msgpack/predef/os.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/
x Data-MessagePack-1.00/include/msgpack/predef/detail/endian_compat.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/os_detected.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/test.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/_exception.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/_cassert.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/comp_detected.h
x Data-MessagePack-1.00/include/msgpack/predef/detail/platform_detected.h
x Data-MessagePack-1.00/include/msgpack/predef/architecture.h
x Data-MessagePack-1.00/include/msgpack/predef/library.h
x Data-MessagePack-1.00/include/msgpack/predef/platform/
x Data-MessagePack-1.00/include/msgpack/predef/platform/windows_store.h
x Data-MessagePack-1.00/include/msgpack/predef/platform/mingw.h
x Data-MessagePack-1.00/include/msgpack/predef/platform/windows_desktop.h
x Data-MessagePack-1.00/include/msgpack/predef/platform/windows_runtime.h
x Data-MessagePack-1.00/include/msgpack/predef/platform/windows_phone.h
x Data-MessagePack-1.00/include/msgpack/unpack_template.h
x Data-MessagePack-1.00/include/msgpack.h
x Data-MessagePack-1.00/xshelper.h
x Data-MessagePack-1.00/inc/
x Data-MessagePack-1.00/inc/Module/
x Data-MessagePack-1.00/inc/Module/Install/
x Data-MessagePack-1.00/inc/Module/Install/Base.pm
x Data-MessagePack-1.00/inc/Module/Install/AuthorTests.pm
x Data-MessagePack-1.00/inc/Module/Install/Fetch.pm
x Data-MessagePack-1.00/inc/Module/Install/Can.pm
x Data-MessagePack-1.00/inc/Module/Install/WriteAll.pm
x Data-MessagePack-1.00/inc/Module/Install/Win32.pm
x Data-MessagePack-1.00/inc/Module/Install/Makefile.pm
x Data-MessagePack-1.00/inc/Module/Install/XSUtil.pm
x Data-MessagePack-1.00/inc/Module/Install/Metadata.pm
x Data-MessagePack-1.00/inc/Module/Install.pm
x Data-MessagePack-1.00/README
x Data-MessagePack-1.00/lib/
x Data-MessagePack-1.00/lib/Data/
x Data-MessagePack-1.00/lib/Data/MessagePack/
x Data-MessagePack-1.00/lib/Data/MessagePack/Unpacker.pod
x Data-MessagePack-1.00/lib/Data/MessagePack/Boolean.pm
x Data-MessagePack-1.00/lib/Data/MessagePack/PP.pm
x Data-MessagePack-1.00/lib/Data/MessagePack.pm
x Data-MessagePack-1.00/.travis.yml
x Data-MessagePack-1.00/t/
x Data-MessagePack-1.00/t/09_stddata.t
x Data-MessagePack-1.00/t/11_stream_unpack3.t
x Data-MessagePack-1.00/t/03_stream_unpack.t
x Data-MessagePack-1.00/t/20_de.t
x Data-MessagePack-1.00/t/16_unpacker_for_larges.t
x Data-MessagePack-1.00/t/10_splitted_bytes.t
x Data-MessagePack-1.00/t/02_unpack.t
x Data-MessagePack-1.00/t/19_utf8_property.t
x Data-MessagePack-1.00/t/17_canonical.t
x Data-MessagePack-1.00/t/21_dirty_float.t
x Data-MessagePack-1.00/t/06_stream_unpack2.t
x Data-MessagePack-1.00/t/04_invert.t
x Data-MessagePack-1.00/t/14_invalid_data.t
x Data-MessagePack-1.00/t/23-empty-key.t
x Data-MessagePack-1.00/t/data.pl
x Data-MessagePack-1.00/t/std/
x Data-MessagePack-1.00/t/std/cases.mpac
x Data-MessagePack-1.00/t/std/cases_compact.mpac
x Data-MessagePack-1.00/t/50_leaktrace.t
x Data-MessagePack-1.00/t/01_pack.t
x Data-MessagePack-1.00/t/00_compile.t
x Data-MessagePack-1.00/t/18_new_interface.t
x Data-MessagePack-1.00/t/Util.pm
x Data-MessagePack-1.00/t/13_booleans.t
x Data-MessagePack-1.00/t/22_pid_pack_unpack.t
x Data-MessagePack-1.00/t/24_tied.t
x Data-MessagePack-1.00/t/15_utf8.t
x Data-MessagePack-1.00/t/05_preferred_int.t
x Data-MessagePack-1.00/t/07_break.t
x Data-MessagePack-1.00/t/08_cycle.t
x Data-MessagePack-1.00/t/40_threads.t
x Data-MessagePack-1.00/t/12_stream_unpack4.t
x Data-MessagePack-1.00/META.yml
x Data-MessagePack-1.00/Changes
x Data-MessagePack-1.00/xt/
x Data-MessagePack-1.00/xt/leaks/
x Data-MessagePack-1.00/xt/leaks/stream.t
x Data-MessagePack-1.00/xt/leaks/normal.t
x Data-MessagePack-1.00/xt/99_pod.t
x Data-MessagePack-1.00/.gitmodules
x Data-MessagePack-1.00/xs-src/
x Data-MessagePack-1.00/xs-src/unpack.c
x Data-MessagePack-1.00/xs-src/pack.c
x Data-MessagePack-1.00/xs-src/MessagePack.xs
x Data-MessagePack-1.00/MANIFEST
x Data-MessagePack-1.00/Makefile.PL
x Data-MessagePack-1.00/LICENSE
Entering Data-MessagePack-1.00
Checking configure dependencies from META.yml
Checking if you have ExtUtils::ParseXS 3.18 ... Yes (3.40)
Checking if you have Devel::PPPort 3.19 ... Yes (3.52)
Checking if you have File::Copy::Recursive 0 ... Yes (0.44)
Checking if you have ExtUtils::MakeMaker 6.59 ... Yes (7.36)
Running Makefile.PL
Configuring Data-MessagePack-1.00 ... Writing ppport.h
Checking if your kit is complete...
Looks good
Generating a gmake-style Makefile
Writing Makefile for Data::MessagePack
Writing MYMETA.yml and MYMETA.json
OK
Checking dependencies from MYMETA.json ...
Checking if you have File::Copy::Recursive 0 ... Yes (0.44)
Checking if you have Devel::PPPort 3.19 ... Yes (3.52)
Checking if you have Test::More 0.94 ... Yes (1.302164)
Checking if you have ExtUtils::ParseXS 3.18 ... Yes (3.40)
Checking if you have XSLoader 0.02 ... Yes (0.30)
Checking if you have ExtUtils::MakeMaker 0 ... Yes (7.36)
Checking if you have Test::Requires 0 ... Yes (0.10)
Building and testing Data-MessagePack-1.00 ... cp lib/Data/MessagePack/Unpacker.pod blib\lib\Data\MessagePack\Unpacker.pod
cp lib/Data/MessagePack.pm blib\lib\Data\MessagePack.pm
cp lib/Data/MessagePack/Boolean.pm blib\lib\Data\MessagePack\Boolean.pm
cp lib/Data/MessagePack/PP.pm blib\lib\Data\MessagePack\PP.pm
Running Mkbootstrap for MessagePack ()
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Command -e chmod -- 644 "MessagePack.bs"
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Command::MM -e cp_nonempty -- MessagePack.bs blib\arch\auto\Data\MessagePack\MessagePack.bs 644
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" "D:\devel\perl\perl\lib\ExtUtils/xsubpp"  -typemap D:\devel\perl\perl\lib\ExtUtils\typemap  xs-src/MessagePack.xs > xs-src/MessagePack.xsc
Please specify prototyping behavior for MessagePack.xs (see perlxs manual)
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Command -e mv -- xs-src/MessagePack.xsc xs-src/MessagePack.c
gcc -c  "-I." "-Iinclude" "-I." -s -O2 -DWIN32 -DWIN64 -DCONSERVATIVE -D__USE_MINGW_ANSI_STDIO -DPERL_TEXTMODE_SCRIPTS -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DUSE_PERLIO -fwrapv -fno-strict-aliasing -mms-bitfields -Wall -W -Wno-comment -s -O2   -DVERSION=\"1.00\" -DXS_VERSION=\"1.00\" -o xs-src/MessagePack.o  "-ID:\devel\perl\perl\lib\CORE"  -DUSE_PPPORT xs-src/MessagePack.c
gcc -c  "-I." "-Iinclude" "-I." -s -O2 -DWIN32 -DWIN64 -DCONSERVATIVE -D__USE_MINGW_ANSI_STDIO -DPERL_TEXTMODE_SCRIPTS -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DUSE_PERLIO -fwrapv -fno-strict-aliasing -mms-bitfields -Wall -W -Wno-comment -s -O2   -DVERSION=\"1.00\" -DXS_VERSION=\"1.00\" -o xs-src/pack.o  "-ID:\devel\perl\perl\lib\CORE"  -DUSE_PPPORT xs-src/pack.c
gcc -c  "-I." "-Iinclude" "-I." -s -O2 -DWIN32 -DWIN64 -DCONSERVATIVE -D__USE_MINGW_ANSI_STDIO -DPERL_TEXTMODE_SCRIPTS -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DUSE_PERLIO -fwrapv -fno-strict-aliasing -mms-bitfields -Wall -W -Wno-comment -s -O2   -DVERSION=\"1.00\" -DXS_VERSION=\"1.00\" -o xs-src/unpack.o  "-ID:\devel\perl\perl\lib\CORE"  -DUSE_PPPORT xs-src/unpack.c
xs-src/unpack.c: In function 'template_callback_root':
xs-src/unpack.c:105:55: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE SV* template_callback_root(unpack_user* u PERL_UNUSED_DECL)
                                          ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_UV':
xs-src/unpack.c:166:53: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_UV(unpack_user* u PERL_UNUSED_DECL, UV const d, SV** o)
                                        ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_IV':
xs-src/unpack.c:175:53: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_IV(unpack_user* u PERL_UNUSED_DECL, IV const d, SV** o)
                                        ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_double':
xs-src/unpack.c:196:57: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_double(unpack_user* u PERL_UNUSED_DECL, double d, SV** o)
                                            ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_nil':
xs-src/unpack.c:204:54: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_nil(unpack_user* u PERL_UNUSED_DECL, SV** o)
                                         ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_true':
xs-src/unpack.c:211:55: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_true(unpack_user* u PERL_UNUSED_DECL, SV** o)
                                          ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_false':
xs-src/unpack.c:217:56: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_false(unpack_user* u PERL_UNUSED_DECL, SV** o)
                                           ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_array':
xs-src/unpack.c:223:56: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_array(unpack_user* u PERL_UNUSED_DECL, unsigned int n, SV** o)
                                           ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_array_item':
xs-src/unpack.c:232:61: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_array_item(unpack_user* u PERL_UNUSED_DECL, SV** c, SV* o)
                                                ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_map':
xs-src/unpack.c:241:54: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_map(unpack_user* u PERL_UNUSED_DECL, unsigned int n, SV** o)
                                         ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_map_item':
xs-src/unpack.c:250:59: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_map_item(unpack_user* u PERL_UNUSED_DECL, SV** c, SV* k, SV* v)
                                              ~~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_str':
xs-src/unpack.c:260:54: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_str(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p, unsigned int l, SV** o)
                                         ~~~~~~~~~~~~~^
xs-src/unpack.c:260:86: warning: unused parameter 'b' [-Wunused-parameter]
 STATIC_INLINE int template_callback_str(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p, unsigned int l, SV** o)
                                                                          ~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_bin':
xs-src/unpack.c:269:86: warning: unused parameter 'b' [-Wunused-parameter]
 STATIC_INLINE int template_callback_bin(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p, unsigned int l, SV** o)
                                                                          ~~~~~~~~~~~~^
xs-src/unpack.c: In function 'template_callback_ext':
xs-src/unpack.c:279:54: warning: unused parameter 'u' [-Wunused-parameter]
 STATIC_INLINE int template_callback_ext(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p PERL_UNUSED_DECL,
                                         ~~~~~~~~~~~~~^
xs-src/unpack.c:279:86: warning: unused parameter 'b' [-Wunused-parameter]
 STATIC_INLINE int template_callback_ext(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p PERL_UNUSED_DECL,
                                                                          ~~~~~~~~~~~~^
xs-src/unpack.c:279:118: warning: unused parameter 'p' [-Wunused-parameter]
 STATIC_INLINE int template_callback_ext(unpack_user* u PERL_UNUSED_DECL, const char* b PERL_UNUSED_DECL, const char* p PERL_UNUSED_DECL,
                                                                                                          ~~~~~~~~~~~~^
xs-src/unpack.c:280:54: warning: unused parameter 'l' [-Wunused-parameter]
                                         unsigned int l PERL_UNUSED_DECL, SV** o PERL_UNUSED_DECL)
                                         ~~~~~~~~~~~~~^
xs-src/unpack.c:280:79: warning: unused parameter 'o' [-Wunused-parameter]
                                         unsigned int l PERL_UNUSED_DECL, SV** o PERL_UNUSED_DECL)
                                                                          ~~~~~^
In file included from xs-src/unpack.c:286:
include/msgpack/unpack_template.h: In function 'template_execute':
include/msgpack/unpack_template.h:250:17: warning: this statement may fall through [-Wimplicit-fallthrough=]
                 ++p;
                 ^~~
include/msgpack/unpack_template.h:252:13: note: here
             default:
             ^~~~~~~
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Mksymlists \
     -e "Mksymlists('NAME'=>\"Data::MessagePack\", 'DLBASE' => 'MessagePack', 'DL_FUNCS' => {  }, 'FUNCLIST' => [], 'IMPORTS' => {  }, 'DL_VARS' => []);"
g++.exe MessagePack.def -o blib\arch\auto\Data\MessagePack\MessagePack.xs.dll -mdll -s -L"D:\devel\perl\perl\lib\CORE" -L"D:\devel\perl\c\lib" xs-src/MessagePack.o xs-src/pack.o xs-src/unpack.o   "D:\devel\perl\perl\lib\CORE\libperl530.a" -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lws2_32 -lmpr -lwinmm -lversion -lodbc32 -lodbccp32 -lcomctl32 -Wl,--enable-auto-image-base
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Command -e chmod -- 755 blib\arch\auto\Data\MessagePack\MessagePack.xs.dll
"D:\devel\perl\perl\bin\perl.exe" "-Iinc" -MExtUtils::Command::MM -e cp_nonempty -- MessagePack.bs blib\arch\auto\Data\MessagePack\MessagePack.bs 644
"D:\devel\perl\perl\bin\perl.exe" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'inc', 'blib\lib', 'blib\arch')" t/*.t
t/00_compile.t .............. # Testing Data::MessagePack/1.00 (XS)
# byteoder: 12345678, ivsize=8
t/00_compile.t .............. ok
t/01_pack.t ................. ok
t/02_unpack.t ............... ok
t/03_stream_unpack.t ........ ok
t/04_invert.t ............... ok
t/05_preferred_int.t ........ 1/30
#   Failed test 'dump (?^:^(c4 15 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 30 33 34|c4 18 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 34 32 31 65 2b 30 33 34)$)'
#   at t/05_preferred_int.t line 23.
#                   'c4 14 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 33 34'
#     doesn't match '(?^:^(c4 15 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 30 33 34|c4 18 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 34 32 31 65 2b 30 33 34)$)'

#   Failed test 'dump (?^:^(c4 16 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 30 33 35|c4 19 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 37 39 33 36 65 2b 30 33 35))'
#   at t/05_preferred_int.t line 23.
#                   'c4 15 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 33 35'
#     doesn't match '(?^:^(c4 16 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 30 33 35|c4 19 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 37 39 33 36 65 2b 30 33 35))'

#   Failed test 'dump (?^:^(b5 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 30 33 34|b8 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 34 32 31 65 2b 30 33 34)$)'
#   at t/05_preferred_int.t line 32.
#                   'b4 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 33 34'
#     doesn't match '(?^:^(b5 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 65 2b 30 33 34|b8 38 2e 33 30 37 36 37 34 39 37 33 36 35 35 37 32 34 32 31 65 2b 30 33 34)$)'

#   Failed test 'dump (?^:^(b6 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 30 33 35|b9 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 37 39 33 36 65 2b 30 33 35))'
#   at t/05_preferred_int.t line 32.
#                   'b5 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 33 35'
#     doesn't match '(?^:^(b6 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 38 65 2b 30 33 35|b9 2d 36 2e 36 34 36 31 33 39 39 37 38 39 32 34 35 37 39 33 36 65 2b 30 33 35))'
# Looks like you failed 4 tests of 30.
t/05_preferred_int.t ........ Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/30 subtests
t/06_stream_unpack2.t ....... ok
t/07_break.t ................ ok
t/08_cycle.t ................ ok
t/09_stddata.t .............. ok
t/10_splitted_bytes.t ....... ok
t/11_stream_unpack3.t ....... ok
t/12_stream_unpack4.t ....... ok
t/13_booleans.t ............. ok
t/14_invalid_data.t ......... ok
t/15_utf8.t ................. ok
t/16_unpacker_for_larges.t .. ok
t/17_canonical.t ............ ok
t/18_new_interface.t ........ ok
t/19_utf8_property.t ........ ok
t/20_de.t ................... ok
t/21_dirty_float.t .......... ok
t/22_pid_pack_unpack.t ...... ok
t/23-empty-key.t ............ ok
t/24_tied.t ................. ok
t/40_threads.t .............. ok
t/50_leaktrace.t ............ ok

Test Summary Report
-------------------
t/05_preferred_int.t      (Wstat: 1024 Tests: 30 Failed: 4)
  Failed tests:  13, 18, 23, 26
  Non-zero exit status: 4
t/23-empty-key.t          (Wstat: 0 Tests: 3 Failed: 0)
  TODO passed:   1-2
Files=27, Tests=575, 18 wallclock secs ( 0.42 usr +  0.17 sys =  0.59 CPU)
Result: FAIL
Failed 1/27 test programs. 4/575 subtests failed.
gmake: *** [Makefile:989: test_dynamic] Error 255
FAIL
! Installing Data::MessagePack failed. See C:\Users\zdm\AppData\Local\Temp\.cpanm\work\1558802524.364\build.log for details. Retry with --force to force install it.
```

**Comment by zdm** [comment]
The same problem under perl 5.30.1.

**Comment by twata1** [comment]
Hello,

Same here. Windows 7, Strawberry Perl 5.30.0 32bit.
I can't install Text::Xslate smoothly because Data::MessagePack 1.0 installation fails.

Here is a build.log.
[build.log](https://github.com/msgpack/msgpack-perl/files/3995062/build.log)

**Comment by syohex** [comment]
I fixed this issue by #45. Please check latest version, `1.01`
